package.json

```diff
@@ -22,6 +22,7 @@
     "vega": "5.4.0",
     "vega-lite": "3.3.0",
     "vega-embed": "4.2.0",
+    "@jupyter-widgets/base": "1.2.5",
```

Comment:
Why not the latest versions 2.x?

Reply:
Only because `1.2.5` was the latest version approx two weeks ago, when I last updated this PR. In hindsight using a lower version bound (or no bound at all) would have been a better choice.

Reply:
Can I update to 2.x without problems?

Reply:
Yes. 

I locally replaced the version bound with `"@jupyter-widgets/base": "^2.0.0"`, which results in an installed version of `2.0.1`. The extension builds without issue and the notebooks executes as expected.